app/controllers/home_controller.rb

```diff
@@ -1,4 +1,6 @@
 class HomeController < ApplicationController
   def index
+    @card = Card.random_need_to_review
+    render 'random_card'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.